bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -546,7 +546,7 @@ spec:
                               format: int64
                               type: integer
                             garbage-collection-frequency:
-                              description: How long to wait by default before backups can be garbage collected. (default 720h0m0s)
+                              description: How long to wait by default before backups can be garbage collected. (gc-controller runs its reconciliation loop every hour by default)
```

Comment:
This fixes the duration problem, but the description text is wrong. This isn't how long to wait but how frequently to run, and it has nothing to do with defaults. Best to replace "How long to wait by default before backups can be garbage collected" with "How frequently Velero checks for expired backups to delete." or something similar

Reply:
sure

Reply:
@sseago Can you please take a look ?